homeassistant/components/drop_connect/translations/en.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
This file can be removed btw

Reply:
Is this because strings.json is assumed to be English?  I saw quite a few integrations that had both files.

Reply:
When you generate translations it outputs to en.json and when you run HA locally it uses the en.json. So yes, you will see this file in every integration on your file system, but when you look in the git repo, you will see none :)

Reply:
I understand now.  I read the translations documentation but overlooked the important bit at the end about generating the translations for testing.  The file was removed in 140a97999773785ca4b914980204c5a9fad63d97.